### PR TITLE
Updated chrome driver for tests

### PR DESCRIPTION
**Pull request by roienatan**
resolves: https://github.com/daostack/alchemy/issues/2092

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,11 @@ before_install:
 addons:
   chrome: stable
 
+before_script:
+  - LATEST_CHROMEDRIVER_VERSION=`curl -s "https://chromedriver.storage.googleapis.com/LATEST_RELEASE"`
```
Comment (dkent600) [inline .travis.yml:18]: Where is LATEST_CHROMEDRIVER_VERSION used?

**Review by orenyodfat — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (roienatan) [reply .travis.yml:18]: In the next line :)

Comment (dkent600) [reply .travis.yml:18]: How does it sync with the changes in wdio.conf.js?

Comment (roienatan) [reply .travis.yml]: For now, I removed it since I'm not sure if it works properly. Will check it next time.
Thanks!

**Pull request merged**